### ISSUE TITLE
Settings: Button for calibration

### ISSUE THREAD

**Issue by drjakob** [roschaefer/story.board#164]
As member of service team
I want to have a button to start calibration of reporter.box
because easy calibration is essential for using the box.

**Comment by drjakob** [comment]
@kabel42: "Kalibrieren sollte jetzt tun, inklusive Anzeige auf dem Display: https://www.dropbox.com/s/qthypehyx35erp1/IMG_20160925_211033.jpg?dl=0 Jetzt brauchen wir nur noch einen Button im Webinterface..."

**Comment by roschaefer** [comment]
OK, I still don't know how the calibration should actually work. I will now implement two attributes on a `Sensor` for highest and lowest value. The user will trigger the start and the end of the calibration. During the calibration, the highest and lowest value will be stored for the sensor. I hope this will be enough at this time.

**Comment by kabel42** [comment]
I thought we had a button that told the Box to start calibration by calling:
https://github.com/kabel42/reporter.box/blob/master/Reporter-Box.ino#L155

**Comment by drjakob** [comment]
Yes, but I learnt that we have to open it up for manual calibration because gassensors are hard to calibrate automatically.
